src/dft/backends/mklcpu/backward.cpp

```diff
@@ -152,11 +152,12 @@ ONEMKL_EXPORT void compute_backward(descriptor_type &desc, sycl::buffer<input_ty
 
     cpu_queue.submit([&](sycl::handler &cgh) {
         auto desc_acc = mklcpu_desc_buffer.template get_access<sycl::access::mode::read>(cgh);
-        auto in_acc = in.template get_access<sycl::access::mode::read_write>(cgh);
+        auto in_acc = in.template get_access<sycl::access::mode::read>(cgh);
         auto out_acc = out.template get_access<sycl::access::mode::write>(cgh);
 
         detail::host_task<class host_kernel_back_outofplace>(cgh, [=]() {
-            DFT_ERROR status = DftiComputeBackward(desc_acc[detail::DIR::bwd], in_acc.get_pointer(),
+            auto inPtr = const_cast<input_type*>(in_acc.get_pointer());
```

Comment:
The variables use snake_case, can you make them consistent?

Reply:
fixed in 4632e61b2cdbdebb6a63ce986941885e106b684d